.codespellrc

```diff
@@ -1,3 +1,3 @@
 [codespell]
-ignore-words-list = blong, belong, afterall, som, assistent
+ignore-words-list = blong, belong, afterall, som, assistent, crasher
```

Comment:
I don't think "belong" needs to be in this list in order to whitelist "blong" - these were both added in 0ae026e19798ce9c2c5767621f2fedacb5e7af33

Reply:
seems to pass without it yeah